### DISCUSSION TITLE
Amstrad 1512/ 1610/ 3086

### DISCUSSION THREAD

**Discussion by brendan1495** [willyvmm/mouSTer#45] [Ideas]
Support for the Amstrad 1512/ 1610/ 3086 would be great as mice for these machines are very difficult to get hold of

**Comment by willyvmm** [comment]
I've checked the schematic and for me it looks like ATARI ST like pinout:
![image](https://github.com/willyvmm/mouSTer/assets/24528286/68ed4f6c-6bad-41e9-8b29-5cd2e4d599cc)
Is it working with Atari ST mouse?

Unfortunately i have no access to any Amstrad PC  to verify that ...

**Reply by brendan1495** [reply]
I have an ST mouse that I have not tried, I will test it and let you know if it works on the Amstrad
